hasher-matcher-actioner/terraform/fetcher/main.tf

```diff
@@ -201,7 +201,7 @@ resource "aws_iam_role_policy_attachment" "fetcher_trigger" {
 ### Config storage ###
 
 resource "aws_dynamodb_table" "threatexchange_config" {
-  name         = "${var.prefix}-ThreatExchangeConfig"
+  name         = "${var.prefix}ThreatExchangeConfig"
```

Comment:
Why this change?

Reply:
`jq` was doing weird things with JSON keys that had `-` in them. COuld figure out how to escape correctly so this seemed easiest